backend/src/test/java/com/speakfluid/backend/entities/ImageStepTests.java

```diff
@@ -0,0 +1,88 @@
+package com.speakfluid.backend.entities;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Tests for the ImageStep class.
+ *
+ * @author  Zoey Zhang
+ * @version 1.0
+ * @since   2022-11-26
+ */
+class ImageStepTests extends TalkStepTest {
+    static ImageStep imageStep = new ImageStep();
+    static Dialogue<WozMessage> d1;
+    static Dialogue<WozMessage> d2;
+    static Dialogue<WozMessage> d3;
+    static ArrayList<WozMessage> botMsg1;
+    static ArrayList<WozMessage> userMsg1;
+    static ArrayList<WozMessage> botMsg2;
+    static ArrayList<WozMessage> userMsg2;
+    static ArrayList<WozMessage> userMsg3;
+    static WozMessage m1 = new WozMessage("text", "Hi, here are the locations of our stores");
+    static WozMessage m2 = new WozMessage("text", "What else can I help you?");
+    static WozMessage m3 = new WozMessage("text", "Hi,");
+    static WozMessage m4 = new WozMessage("text", "Can I have the direction to the store?");
+    static WozMessage m5 = new WozMessage("text", "no");
+
+
+
+    @BeforeAll
+    public static void setUp(){
+        botMsg1 = new ArrayList<>(Arrays.asList(m1,m2));
+        userMsg1 = new ArrayList<>(Arrays.asList(m3, m4));
+        d1 = new Dialogue<>(botMsg1, userMsg1);
+        botMsg2 = new ArrayList<>(Arrays.asList(m1,m2));
+        userMsg2 = new ArrayList<>();
+        d2 = new Dialogue<>(botMsg2, userMsg2);
+        userMsg3 = new ArrayList<>(Arrays.asList(m5));
+        d3 = new Dialogue<>(botMsg2, userMsg3);
+
+    }
+
+    @Test
+    void testGetStepName() {
+        assertEquals("Image", imageStep.getStepName());
+    }
+
+    @Test
+    void testGetMaxScore() {
+        assertEquals(18.0, imageStep.getMaxScore());
+    }
+
+    @Test
+    void testGetScoreAccumulator() {
+        assertEquals(26.0, imageStep.getScoreAccumulator());
```

Comment:
why would the scoreAccumulator be 26.0 here? no runAnalysis() is run right

Reply:
That is an issue I am tackling, somehow the scoreaccumulator was carried over from other tests.